zuul.d/jobs.yaml

```diff
@@ -6,3 +6,14 @@
       asciidoc_available: true
       # This var is specific to ci-framework docs directory using sphinx
       doc_available: false
+
+- job:
+    name: data-plane-adoption-osp-17-to-extracted-crc-minimal-no-ceph
+    parent: data-plane-adoption-osp-17-to-extracted-crc
+    vars:
+      use_ceph: "false"
+      dpa_test_suite: "test-minimal"
+    # TODO jgilaber: decide on when we should run this variant
```

Comment:
I think we'd want this job also for Glance and Nova changes? IIRC Compute team wanted a non-Ceph job too. cc @SeanMooney @gibizer @bogdando

Reply:
(Anyway i think that can be added in a subsequent PR.)

Reply:
yes i would generally say that we should run both the ceph and non ceph version on all change unless there is a strong reason not too.

i would propose one adoption job using ceph for nova/glance/cinder
and one without ceph with glance using the file or swift backend, cinder using lvm or nfs and nova using qcow or raw.

Reply:
+1 I agree to have both jobs (w/ and w/o ceph). If no ceph, `Glance` will be configured (by TripleO) w/ `Swift` as a backend in the src cloud. @SeanMooney with this patch you don't need any other change in the `Glance` config (unless you have special needs or you need a particular backends combination).